src/modules/simulation/gz_bridge/GZBridge.cpp

```diff
@@ -582,6 +582,22 @@ void GZBridge::poseInfoCallback(const gz::msgs::Pose_V &msg)
 
 			local_position_groundtruth.timestamp = timestamp;
 			_lpos_ground_truth_pub.publish(local_position_groundtruth);
+
+
+			airflow_s airflow_groundtruth;
+			airflow_groundtruth.timestamp = timestamp;
+			// Body-relateive air velocity
+			matrix::Dcmf Rib(matrix::Quatf{(float)q_nb.W(), (float)q_nb.X(), (float)q_nb.Y(), (float)q_nb.Z()});
+			matrix::Vector3f velocity_f = matrix::Vector3f{(float)velocity(0), (float)velocity(1), (float)velocity(2)};
+			matrix::Vector3f body_velocity = Rib.transpose() * velocity_f;
```

Comment:
```suggestion
			matrix::Vector3f body_velocity = matrix::Quatf(vehicle_attitude_groundtruth.q).rotateVectorInverse(velocity_f);
```

And can't you just use `matrix::Vector3f(velocity)` to convert from double to float?